frontend/src/locales/da/input_fields.json

```diff
@@ -17,6 +17,8 @@
   "name": "Navn",
   "new_name": "Nyt navn",
   "org_unit_type": "@:shared.org_unit_type",
+  "user_key": "Enhedsnummer",
```

Comment:
Rename this to something more specific to org unit user keys - we wouldn't want _every_ potential user_key field to be called "Enhedsnummer"